### DISCUSSION TITLE
Shields.io description and CC0 license

### DISCUSSION THREAD

**Discussion by awwwis** [badges/shields#5510] [Q&A] [answered]
Hi, first of all, I want to say this project is awesome. 
I just notice that the description for Shields.io that is on meta is described as:
[Shields.io: Quality metadata badges for open source projects](https://github.com/badges/shields/blob/0d8a2d5ca0a93c9ffb8456212771d0762bee96f5/frontend/components/meta.tsx#L14)
And License wise it is assigned with CC0 which allows private use (non open source projects).
Since there seems to be some contradiction there, I thought I'll mention it and ask why?

**Comment by calebcartwright** [comment, marked as answer]
> Hi, first of all, I want to say this project is awesome.

Thanks :heart: 

> Since there seems to be some contradiction there, I thought I'll mention it and ask why?

With the obligatory disclaimer that I'm not a lawyer specializing in open source licensing.. there's a difference between a description of the _capabilities_ on a site/service like Shields.io vs. the licensing of the underlying source code that's shared/distributed and the contributions to said source code. 

The description on the site is such because generally speaking Shields.io can only be used to generate badges for public, open projects for authorization/authentication reasons. Regardless of the OSS License applied to source, the Shields.io service is never going to be authorized to access private repo data on GitHub, private builds on TravisCI, etc. just like how neither I nor the other maintainers can access your private projects. 

That's not always obvious to users though (lots of "why can't Shields create a badge for my private project/repo/etc." questions), so the wording in the short description is intended to help convey that message.

**Comment by SuvaJana** [comment]
Hello @calebcartwright I saw this discussion and I have a quick question regarding this.  Could you please tell me what is the license for the output generated while using https://img.shields.io/badge/<LABEL>-<MESSAGE>-<COLOR> for example. Is it still under CCO? or this service has any terms of service?

Your time is much appreciated.  Many thanks.
